standard/toolchain-android32.cmake

```diff
@@ -1,4 +1,11 @@
-if(NOT DEFINED CMAKE_ANDROID_ARCH_ABI)
-  set(CMAKE_ANDROID_ARCH_ABI armeabi)
-endif()
+set(CMAKE_ANDROID_ARCH_ABI armeabi)
+set(CMAKE_ANDROID_ARM_MODE ON)
```

Comment:
Could keep `CMAKE_ANDROID_ARCH_ABI` conditional on whether it's not already defined, as @yeswalrus had it? I think the idea is that it should be easy to switch to another 32-bit architecture like armv7-a later if we decide to.

Are `CMAKE_ANDROID_ARM_MODE` and `-mfpu=neon` necessary here? Looks like we weren't using it before, and I checked the autowiring build scripts too.

Reply:
Ah..good point. I will recover the condition. As for -mfpu=neon, we never used it as you said. I think we never worked on armv5te as mentioned by @yeswalrus. armv5 is very old. I think we can get rid of it. What do you think, @yeswalrus ?